src/Microsoft.NET.Test.Sdk.props

```diff
@@ -11,8 +11,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ProjectCapability Include="TestContainer" />
```

Comment:
@davkean this was discussed earlier. Pl refer comments on feb 12th. 
@bradwilson it will be good to add this projectcapability in the xunit.runner.visualstudio nuget package. Can this be done for 2.3 release?